docs/landing/feedback.html

```diff
@@ -52,15 +52,16 @@
         document.addEventListener("DOMContentLoaded", () => {
             const statsURL = "https://sigrid-says.com/usage/matomo.php";
             const params = new URLSearchParams(window.location.search);
+            const timestamp = new Date().getTime();
```

Comment:
Does this result in a URL-friendly timestamp, preferably in ISO 8601 format? And should it be normalized to UTC (this might run anywhere on our planet in a CI/CD environment).

Reply:
It returns the timestamp as an integer. We don't actually use the timestamp for anything, this `t=` is a convention to "break through" proxies that might otherwise cache GET requests to images, even when the request sets the correct response headers to avoid caching.